pkgs/development/tools/clang-tools/wrapper

```diff
@@ -13,8 +13,19 @@ buildcpath() {
   echo $path
 }
 
-export CPATH=${CPATH}${CPATH:+':'}$(buildcpath ${NIX_CFLAGS_COMPILE})
-export CPATH=${CPATH}${CPATH:+':'}@libc_includes@
-export CPLUS_INCLUDE_PATH=${CPATH}${CPATH:+':'}@libcpp_includes@
+setpaths() {
+  local nix_cpath=$(buildcpath ${NIX_CFLAGS_COMPILE})
+  export CPATH=${CPATH}${CPATH:+':'}${nix_cpath}
+
+  if [[ -f "$NIX_CC/nix-support/cpath" ]]; then
+    export CPATH=${CPATH}${CPATH:+':'}$(< "$NIX_CC/nix-support/cpath")
+  fi
+
+  if [[ -f "$NIX_CC/nix-support/cxxpath" ]]; then
+    export CPLUS_INCLUDE_PATH=${CPLUS_INCLUDE_PATH}${CPLUS_INCLUDE_PATH:+':'}$(< "$NIX_CC/nix-support/cxxpath")
```

Comment:
If I am not mistaking this should also fix https://github.com/NixOS/nixpkgs/pull/73345#commitcomment-36463293